### ISSUE TITLE
Pet and Currency tabs are missing under my Character tab

### ISSUE THREAD

**Issue by OHuntard** [dalaranwow/dalaran-wow#4860]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1.  Instant level 70 my character with your paid service
2.  Changed faction to horde
3.  Attempted to place my mount on my action bar by clicking character tab then pets/mounts and they are not there, they are missing. Typed Shift/P and it does nothing at all, very frustrating. I included a picture to show them missing , please fix this asap 👍 
4. Also on my Paladin, Aleckzander I bought a Mount (swift tiger) it says nothing in the store to describe which faction can use what items , I bought it and then it says can not use after I give my points , not happy about that, please make thumbnails work so people will know what they can or can not use, makes sense right? please fix this for me too 👍 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![wowscrnshot_082717_222553](https://user-images.githubusercontent.com/31396288/29757045-e2f218c2-8b76-11e7-8858-0e9099cf6d35.jpg)

**Comment by TeridanDW** [comment]
Hello @OHuntard 
It's normal for these tabs to be missing as you used instant 70 feature. They will appear once you learn 1 pet/mount or get 1 emblem/currency. As for the "swift tiger" i will asume that happened when the site still uses openwow, now it's wowhead.
